mParticle-Apptimize/MPKitApptimize.m

```diff
@@ -75,6 +93,9 @@ - (void)start {
     NSDictionary *options = [self buildApptimizeOptions];
     void(^start_block)(void) = ^{
         [Apptimize startApptimizeWithApplicationKey:self.configuration[APP_MP_KEY] options:options];
+        if (_pilotTargetingID) {
+            [Apptimize setPilotTargetingID: _pilotTargetingID];
```

Comment:
Why add support for this (setting the id at start up)? It feels a little out of place. There are no other properties that get set this way.

Reply:
Is there a way to take advantage of the options?

Reply:
Apptimize documentation recommends setting the pilotTargetingId after setting up Apptimize

https://apptimize.com/docs/featureflags/installation.html
> We recommend setting the Pilot Targeting ID immediately after calling Apptimize setup()

Reply:
There might be a way to take advantage of the options